website/src/components/Home.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import Dashboard from './dashboard/Dashboard';
+import Landing from './landing/Landing';
+
+class Home extends React.Component {
+   /**
+    * Renders HTML for the home page depending on whether user is
+    * logged in or not.
+    * @returns {HTML}    The HTML of the home page.
+    */
+   render() {
+     const userId = this.props.userInfo.userId;
```

Comment:
I think we should use firebase auth here, in the situation where someone exits the app but doesn't log out, their user name might be persisted but the logout might timeout, which could result in a user that isn't logged in on firebase getting in to the app, which would cause permission errors.